features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Details/index.jsx

```diff
@@ -139,7 +139,7 @@ class Details extends Component {
     render() {
         const redirect_url = '/applications/' + this.props.match.params.application_uuid + '/productionkeys';
 
-        const { classes, theme } = this.props;
+        const { classes, theme, match } = this.props;
```

Comment:
This deconstruction can be moved to the top of the render method and 'const match' be used in 'redirect_url ' and ' <InfoBar applicationId={this.props.match.params.application_uuid}'

Reply:
fixed